styles/standard.json

```diff
@@ -268,8 +268,15 @@
 				"properties": {
 					"razed:railway":"rail",
 					"railway":"razed"
-				}
-			}],
+					
+				}},					
+				{"type": "LineString",
+				"coordinates": [[10,50],[60,50]],
+				"properties": {
+					"dismantled:railway":"rail",
+					"railway":"dismantled"
+				}}			
+			],
```

Comment:
You don't need to duplicate this legend entry. Both variations of tagging (razed and dismantled) are rendered the same. The way you currently did it, leads to to lines at the same location above each other. It has no negative effect on rendering the legend but it is unnecessary.

Reply:
> razed and dismantled are rendered the same.
Really I thought before this change dismanteld would not be renderd

Reply:
okay correction: so I copied this scheme from the main rail lines which alsohave multiple ways of being tagged.

Isn't the "duplicate" needed for if there is no razed but only dismanteled in an area so it still shows up in the legend